src/styles/components/SmartBanner.scss

```diff
@@ -0,0 +1,301 @@
+.smartbanner-show {
+  margin-top: 80px;
+}
+
+.smartbanner-show .smartbanner {
+  display: block;
+}
+
+/** Default **/
+.smartbanner {
+  position: absolute;
+  left: 0;
+  top: 0;
+  display: none;
+  width: 100%;
+  height: 80px;
+  line-height: 80px;
+  font-family: 'Helvetica Neue', sans-serif;
```

Comment:
IMO we should use font that we have used in quran.com. Font family should be either `source sans pro` or `Montserrat` or combination of both.